hbase-server/src/main/java/org/apache/hadoop/hbase/master/MetaFixer.java

```diff
@@ -63,7 +61,7 @@
   MetaFixer(MasterServices masterServices) {
     this.masterServices = masterServices;
     this.maxMergeCount = this.masterServices.getConfiguration().
-        getInt(MAX_MERGE_COUNT_KEY, MAX_MERGE_COUNT_DEFAULT);
+        getInt("hbase.master.metafixer.max.merge.count", 10);
```

Comment:
It was configurable before this change? You could add

  <property>
    <name>hbase.master.metafixer.max.merge.count</name>
    <value>11</value>
    <description>configure the max merge count</description>
  </property>

...
to your hbase-site.xml and away you go?

But maybe you think it deserves mention in the hbase-default.xml so operators can find it easier? If so, I'd suggest a fuller description.

Thanks.

Reply:
Yes if the purpose is to mention this config in `hbase-default.xml`, good to also document it in `hbase-default.adoc` with detailed description.
However, in that case, title of PR doesn't match. As @saintstack said, this is already configurable.

Thanks.